core/src/main/kotlin/renderers/HtmlRenderer.kt

```diff
@@ -1,26 +1,52 @@
 package org.jetbrains.dokka.renderers
 
 import kotlinx.html.*
-import kotlinx.html.stream.appendHTML
+import kotlinx.html.stream.createHTML
+import org.jetbrains.dokka.model.Function
 import org.jetbrains.dokka.pages.*
 import org.jetbrains.dokka.plugability.DokkaContext
+import org.jetbrains.kotlin.utils.addToStdlib.ifNotEmpty
 import java.io.File
-import java.net.URL
 
 open class HtmlRenderer(
     outputWriter: OutputWriter,
     context: DokkaContext
 ) : DefaultRenderer<FlowContent>(outputWriter, context) {
 
-    override fun FlowContent.buildList(node: ContentList, pageContext: PageNode) =
+    private val pageList = mutableListOf<String>()
+
+    private var idCounter = 0
+        get() = ++field
+
+    private fun FlowContent.buildSideMenu(context: PageNode, node: PageNode) {
+        val children = node.children.filter { it !is MemberPageNode }
+        val className = children.ifNotEmpty { "nav$idCounter" }
```

Comment:
Not sure about `ifNotEmpty` as it is in `addToStdlib` package which proved to be unstable in the past. I don't have a strong opinion though

Reply:
I've removed this function call. Also I've refactored this piece of code slightly.